ts/utils/input.ts

```diff
@@ -128,3 +133,15 @@ export function getCreditCardFromState(
 
   return some(card);
 }
+
+/**
+ * This function checks if the cardholder charset is ammitted.
+ * We consider not a valid character an accented character.
+ *
+ * @param cardHolder
+ */
+export const isValidCardHolder = (cardHolder: Option<string>) =>
```

Comment:
```suggestion
export const isValidCardHolder = (cardHolder: Option<string>) : boolean =>
```

Reply:
Solved in [24476f8](https://github.com/pagopa/io-app/pull/2833/commits/24476f86deeb9520699edb91507bd83a646f80bf)